src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -8632,7 +8632,7 @@ static void Main()
 
             Assert.IsType<TupleElementFieldSymbol>(m1Item1);
             Assert.IsType<TupleFieldSymbol>(m2Item1);
-            Assert.IsType<TupleRenamedElementFieldSymbol>(m2a2);
+            Assert.IsType<TupleVirtualElementFieldSymbol>(m2a2);
```

Comment:
Similarly here, I think it would be better to test the internal property than the identity of the type.